smarts/env/gymnasium/wrappers/metric/completion.py

```diff
@@ -122,16 +131,16 @@ def _dist_remainder():
     step: int = 0
 
     def func(
-        road_map: RoadMap, obs: Observation, initial_compl: Completion
+        road_map: RoadMap, obs: Dict[str, Any], initial_compl: Completion
     ) -> Completion:
         nonlocal mean, step
 
-        if obs.events.reached_goal:
+        if obs["events"]["reached_goal"]:
             dist = 0
         else:
-            cur_pos = Point(*obs.ego_vehicle_state.position)
+            cur_pos = Point(*obs["ego_vehicle_state"]["position"])
             # pytype: disable=attribute-error
-            goal_pos = obs.ego_vehicle_state.mission.goal.position
+            goal_pos = Point(*obs["mission"]["goal_position"])
             # pytype: enable=attribute-error
```

Comment:
Probably no need to disable attribute error anymore (i.e., `# pytype: disable=attribute-error`).